### PR TITLE
Fixed rubocop and foodcritic for osl-drupal

### PR DESCRIPTION
**Pull request by cthalmann**
Chefspec tests need to be written for osl-drupal. In order to pass rake, it needs to pass rubocop and foodcritic before it even gets to chefspec. I also ran chef generate in order to add necessary files and changed deprecations.

### CODE REVIEW

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+DEBIAN_8 = {
+  platform: 'debian',
+  version: '8.4'
+}.freeze
+
+ALL_PLATFORMS = [
+  CENTOS_6,
+  CENTOS_7,
+  DEBIAN_8
```
Comment (ramereth) [inline spec/spec_helper.rb]: Please remove Debian as we don't test this platform or use it in this cookbook.

**Comment by ramereth** [comment]
@cthalmann can you please also update the ``.kitchen.yml`` so that it works with our standards?

File: .kitchen.yml [hunk 2]
```diff
@@ -1,14 +1,5 @@
...
 suites:
   - name: default
     run_list:
       - recipe[osl-drupal::default]
-    attributes:
-  - name: drush
-    run_list:
-      - recipe[yum::epel]
-      - recipe[osl-drupal::drush]
-    attributes:
```
Comment (ramereth) [inline .kitchen.yml]: Why are you removing the drush suite?

**Comment by ramereth** [comment]
@cthalmann can you please add the ChefSpec file for the drush suite?

File: .rubocop.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+Metrics/LineLength:
+  Max: 120
+
+Style/IfUnlessModifier:
+  MaxLineLength: 120
+
+Style/WhileUntilModifier:
+  MaxLineLength: 120
+
+Metrics/BlockLength:
```
Comment (fahlmant) [inline .rubocop.yml]: This can actually be removed now

File: .kitchen.yml [hunk 4]
```diff
@@ -3,3 +3,7 @@ suites:
   - name: default
     run_list:
       - recipe[osl-drupal::default]
+  - name: drush
+    run_list:
+      - recipe[yum::epel]
```
Comment (ramereth) [inline .kitchen.yml]: You can remove this from the run list as its included in the recipe.

**Comment by cthalmann** [comment]
!bump patch ~

**Comment by osuosl-manatee** [comment]
Jenkins has merged this PR into `master` and has automatically performed a patch-level version bump to v0.2.2.  Have a nice day!

**Pull request merged**